setup.py

```diff
@@ -3,9 +3,9 @@
 
 setup(
     name='sb-vision',
-    version='1.0',
-    description="Vision system for SourceBots",
-    author="SourceBots",
+    version='0.1.0',
+    description='Vision system for SourceBots',
+    author='SourceBots',
```

Comment:
I suspect these two lines were double quotes intentionally -- there's a convention developing to use double quotes for strings intended for humans and single quotes for strings for machines.